src/directLine.ts

```diff
@@ -439,7 +439,7 @@ export class DirectLine implements IBotConnection {
 
         if (options.pollingInterval && interval < POLLING_INTERVAL_LOWER_BOUND) {
```

Comment:
Also need to protect against non-number. Like this?

```suggestion
         if (options.pollingInterval && typeof options.pollingInterval === 'number') {
           const interval = Math.min(~~options.pollingInterval, POLLING_INTERVAL_LOWER_BOUND);

           if (interval < POLLING_INTERVAL_LOWER_BOUND) {
             console.warn(`DirectLineJS: provided pollingInterval (${options.pollingInterval}) is under lower bound (200ms), using default of 1000ms`);
           } else {
             this.pollingInterval = options.pollingInterval;
           }
         }
```